stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -76,6 +76,8 @@ public interface ChatDomain {
     /** The retry policy for retrying failed requests */
     public var retryPolicy: RetryPolicy
 
+    public val typingUpdates: LiveData<Pair<String, List<User>>>
```

Comment:
`Pair` is a really inconvenient class to use in an API, we should almost always introduce our own class instead. Here, someone observing this `LiveData` would have to write down things like `value.second.size` to get the number of users typing, etc.

If we add our own class to wrap the channel ID and the list of users, it can have well-named properties instead of `first` and `second`.

Reply:
Where do we normally put our model classes? I see `io.getstream.chat.android.client.models`, but nothing similar in the   `stream-chat-android-offfile` project.

Reply:
Well, this touches on the sensitive issue where we should separate our domain models and our DTOs... We have an `entity` package in the `offline` module, which kind of contains models, but some of those are Room entities while some others aren't. Not sure what's the best place to put a new model.

Reply:
Ok. I'll change to a DTO and then we discuss where to place it, move classes is super simple

Reply:
Just create a class that model this info.
We will need to update it later with new "Typing Events" that will perform over threads (It is not implemented yet).
For now, the place to store our models is on the Client module (`io.getstream.chat.android.client.models`) In the future this package could live in a "common" module instead

Reply:
Done, mapped to TypeEvent.